.gitignore

```diff
@@ -77,8 +77,6 @@ local.properties
 *.log
 
 # Build results
-[Dd]ebug/
-[Rr]elease/
```

Comment:
No, let's just add `/[Rr]elease/` and `/[Dd]ebug/` to `.gitignore`. Sorry for lack of consideration.